packages/gatsby-plugin-page-creator/README.md

```diff
@@ -0,0 +1,36 @@
+# gatsby-plugin-page-creator
+
+Plugin to auto create pages using custom directory paths.
```

Comment:
This sentence should make it clear this plugin only creates pages from React components.